doc/DS18S20.md

```diff
@@ -0,0 +1,23 @@
+Dallas Semiconductors DS18S20 temperature sensor
+================================================
+
+Load the one-wire drivers:
```

Comment:
It's another protocol that just requires a single signal wire that does all the read/write/clocking on a single wire (+ ground). It's supported in the Linux kernel through /sys/bus and there's a good write up on https://en.m.wikipedia.org/wiki/1-Wire

Cant get it working on latest kernel on RPi zero though. Definitely have had it working though...